book/website/_toc.yml

```
@@ -348,22 +348,46 @@ parts:
 
   # ===== Guide for Collaboration ========================================
 
-  - file: collaboration/collaboration
+- file: collaboration/collaboration
+  sections:
+  - title: Defining Collaboration Style
+    file: collaboration/defining-style
+    sections:
+      - title: Open source or inner source
+        file: collaboration/defining-style/open-inner-source
+      - title: Defining your community or team
+        file: collaboration/defining-style/defining-community
+      - title: Defining the purpose of an individual interaction
+        file: collaboration/defining-style/defining-purpose
+      - title: Asynchronous or synchronous collaboration
+        file: collaboration/defining-style/async-sync
+      - title: Remote, in person and hybrid collaboration
+        file: collaboration/defining-style/remote-inperson-hybrid
+  - title: Getting Started With GitHub
+    file: collaboration/github-novice
+    sections:
+    - title: Motivation for Using GitHub
+      file: collaboration/github-novice/github-novice-motivation
+    - title: First steps on Github
+      file: collaboration/github-novice/github-novice-firststeps
+    - title: Using more Github features
+      file: collaboration/github-novice/github-novice-features
+    - title: Advanced Github features
+      file: collaboration/github-novice/github-novice-advanced
+  - title: Collaborating on GitHub
+    file: collaboration/github-collaboration
+    sections:
+    - title: README and Project Communication
+      file: collaboration/github/github-readme
+    - title: Roadmapping
+      file: collaboration/github/github-roadmapping
+    - title: Getting Contributors
+      file: collaboration/github/github-contributors
+    - title: Checklist and resources
+      file: collaboration/github/github-resources
+  - title: Maintainers and Reviewers on GitHub
+    file: collaboration/maintain-review
```

```suggestion
    - title: Maintainers and Reviewers on GitHub
      file: collaboration/maintain-review
```
Given that this is specifically about maintenance and review on GitHub, I wonder if it should also be part of the github-collaboration chapter?